### PR TITLE
v1.3: Delete account

### LINKED ISSUE #71: Deleting account(s)
Admin/owner should be able to delete account(s) of staff that are no longer employed.

### PR DESCRIPTION
**Pull request by AZhiKai**
This closes #71

**READY FOR REVIEW!**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/accounts/DeregisterCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.commands.accounts;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```
Comment (HyperionNKJ) [inline src/test/java/seedu/address/logic/commands/accounts/DeregisterCommandTest.java]: Besides this Jupiter import, all else looks good!
I think we can resolve this in v1.4 as there are other Jupiter imports in some of our files.

Comment (AZhiKai) [reply src/test/java/seedu/address/logic/commands/accounts/DeregisterCommandTest.java]: 😧 keep missing that. Will modify later tonight. Thanks!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -308,7 +308,9 @@ public RecordListPanel getRecordListPanel() {
         return recordListPanel;
     }
 
-    public IngredientListPanel getIngredientListPanel() { return ingredientListPanel; }
+    public IngredientListPanel getIngredientListPanel() {
+        return ingredientListPanel;
+    }
```
[the remark this answers is not in the document]
Comment (rebstan97) [reply src/main/java/seedu/address/ui/MainWindow.java]: Right, thanks 👍

**Comment by AZhiKai** [comment]
@rebstan97 yes only when logged in. When i come to implementing command ranking, it can only be executed by the highest rank (i.e. admin). So no, you won't be able to deregister yourself.

File: src/test/java/seedu/address/logic/commands/accounts/DeregisterCommandTest.java (hunk 1 above)
Comment (AZhiKai) [reply src/test/java/seedu/address/logic/commands/accounts/DeregisterCommandTest.java]: Fixed.

**Pull request merged**